themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties

```diff
@@ -0,0 +1,224 @@
+federatedIdentityConfirmLinkMessage=Gebruiker met {0} {1} bestaat al. Hoe wilt u doorgaan?
+federatedIdentityConfirmReauthenticateMessage=Authenticeer als {0} om uw account te koppelen {1}
+confirmLinkIdpReviewProfile=Nalopen profiel
+confirmLinkIdpContinue=Voeg toe aan bestaande account
+
+configureTotpMessage=U moet de Mobile Authenticator configuren om uw account te activeren.
+updateProfileMessage=U moet uw gebruikersprofiel bijwerken om uw account te activeren.
+updatePasswordMessage=U moet uw wachtwoord wijzigen om uw account te activeren.
+verifyEmailMessage=U moet uw e-mailadres verifiëren om uw account te activeren.
+linkIdpMessage=U moet uw e-mailadres verifiëren om uw account te koppelen aan {0}.
+
+emailSentMessage=U zult binnenkort een e-mail ontvangen met verdere instructies.
+emailSendErrorMessage=Het versturen van de e-mail is mislukt, probeer het later opnieuw.
+
+accountUpdatedMessage=Uw account is bijgewerkt.
+accountPasswordUpdatedMessage=Uw wachtwoord is bijgewerkt.
+
+noAccessMessage=Geen toegang
+
+invalidPasswordMinLengthMessage=Ongeldig wachtwoord, de minimumlengte is {0} karakters.
+invalidPasswordMinDigitsMessage=Ongeldig wachtwoord, deze moet minstens {0} cijfers bevatten.
+invalidPasswordMinLowerCaseCharsMessage=Ongeldig wachtwoord, deze moet minstens {0} kleine letters bevatten.
+invalidPasswordMinUpperCaseCharsMessage=Ongeldig wachtwoord, deze moet minstens {0} hoofdletters bevatten.
+invalidPasswordMinSpecialCharsMessage=Ongeldig wachtwoord, deze moet minstens {0} speciale tekens bevatten.
+invalidPasswordNotUsernameMessage=Ongeldig wachtwoord, deze mag niet gelijk zijn aan de gebruikersnaam.
+invalidPasswordRegexPatternMessage=Ongeldig wachtwoord, deze komt niet overeen met opgegeven reguliere expressie(s).
+invalidPasswordHistoryMessage=Ongeldig wachtwoord, deze mag niet gelijk zijn aan een van de laatste {0} wachtwoorden.
+
+failedToProcessResponseMessage=Het verwerken van de respons is mislukt
+httpsRequiredMessage=HTTPS vereist
+realmNotEnabledMessage=Realm niet geactiveerd
+invalidRequestMessage=Ongeldige request
+failedLogout=Afmelden is mislukt
+unknownLoginRequesterMessage=De login requester is onbekend
+loginRequesterNotEnabledMessage=De login requester is niet geactiveerd
+bearerOnlyMessage=Bearer-only applicaties mogen geen browser login initiëren
+standardFlowDisabledMessage=Client is mag geen browserlogin starten met het opgegeven response_type. Standard flow is uitgeschakeld voor de client.
+implicitFlowDisabledMessage=Client is mag geen browserlogin starten met opgegeven response_type. Implicit flow is uitgeschakeld voor de klant.
+invalidRedirectUriMessage=Ongeldige omleiding uri
+unsupportedNameIdFormatMessage=Niet-ondersteunde NameIDFormat
+invalidRequesterMessage=Ongeldige requester
+registrationNotAllowedMessage=Registratie is niet toegestaan
+resetCredentialNotAllowedMessage=Het resetten van de aanmeldgegevens is niet toegestaan
+permissionNotApprovedMessage=Toestemming niet goedgekeurd.
```

Comment:
Not sure what this means. This message also seems to be unused in the code, so no matter.